src/backend/tests/it/utils/assertion.rs

```diff
@@ -19,6 +20,25 @@ pub fn assert_custom_tokens_eq(
     assert_eq!(results_tokens.len(), expected_tokens.len());
 
     for (token, expected) in results_tokens.iter().zip(expected_tokens.iter()) {
-        assert!(expected == token);
+        assert!(expected.token == token.token);
+        assert_eq!(expected.enabled, token.enabled);
+    }
+}
+
+pub fn assert_some_tokens_timestamp<T>(tokens: Vec<T>)
```

Comment:
```suggestion
pub fn assert_some_tokens_timestamp<T>(tokens: &[T])
```
If you pass by ref there is no need to clone.

Reply:
I should have run clippy before submitting the PR.